maestro/README.md

```diff
@@ -116,24 +116,29 @@ maestro run None workflow.yaml
 
 ### Run workflow in local environment with container runtime (e.g. podman) or kubernetes cluster
 
-* Prepare required environments
+* Prepare required environments, running from Bee-Hive top level:
   * Run a local instance of the [Bee Stack](https://github.com/i-am-bee/bee-stack)
-  * `curl` is required for `run` command
-  * for Kubernetes, `kubectl` and `sed` are required
-  * set CONTAINER_CMD, TARGET_IP and BUILD_FLAGS environment variables to adjust maestro.sh script
-    * CONTAINER_CMD: docker, podman. nerdctl, etc..
-    * TARGET_IP: 127.0.0.1:"NodePort of the maestro service" for kubernetes deployment
-    * BUILD_FLAGS: image build command extra flag (e.g. "--namespace k8s.io" for nerdctl)
-
-* Build maestro container image: `./maestro.sh build` in maestro/maestro directory
-  * The built maestro:latest image need to be pushed/placed in the local retistry for kubernetes deployment
-
-* Deploy maestro: `./maestro.sh deploy BEE_API=http://xxx.xxx.xxx.xxx:4000 BEE_API_KEY=sk-proj-testkey`
-  * To deploy to kubernetes cluster, user `deploy-k` instead `deploy` as the first argument
-  * The required environment variables can be provided at the end of command arguments (e.g. `BEE_API=http://192.168.86.45:4000`).
-
-* Prepare agent and workflow definition yaml files in the current directory (e.g. agent.yaml, workflow.yaml)
-
-* Run workflow (directly creating agents and running): `maestro run agents.yaml workflow.yaml`
+  * for Kubernetes, `kubectl`, `kind` are required
```

Comment:
Inlcude links to all external resources since some folks might need to get these setup first